developers/integration.rst

```diff
@@ -212,28 +192,22 @@ addresses.
 Broadcasting InstantSend Transactions
 -------------------------------------
 
-Automatic InstantSend introduces two requirements into the system being
-integrated:
-
-1. The ability to evaluate the number of inputs in a given transaction.
-
-2. The ability to apply an increased fee-level in cases of 5+ inputs.
+Since Dash 0.14.0 established LLMQ on the Dash network, quorums will now
+attempt to lock every valid transaction by default without any
+additional fee or action by the sending wallet or user. A transaction is
+eligible for InstantSend when each of its inputs is considered
+confirmed. This is the case when: 
```

Comment:
Should maybe clarify that it's `or` for conditions below i.e. you either one should be `true` for input to be considered confirmed.

Reply:
```suggestion
confirmed. This is the case when at least one of the following circumstances is true: 
```